modules/src/test/java/eu/itesla_project/modules/contigencies/TapChangeActionTest.java

```diff
@@ -47,7 +47,7 @@ public void testToTask() {
 
         TapChangeAction action = new TapChangeAction("PS1", 2);
         ModificationTask task = action.toTask();
-        task.modify(network);
+        task.modify(network, null);
```

Comment:
Use LocalComputationManager instead of null. Use the LocalComputationManager(Path) constructor configured with an in-memory path (JimFS)